besu/src/test/resources/everything_config.toml

```diff
@@ -51,6 +51,11 @@ rpc-http-cors-origins=["none"]
 rpc-http-authentication-enabled=false
 rpc-http-authentication-credentials-file="none"
 rpc-http-authentication-jwt-public-key-file="none"
+rpc-http-tls-enabled=false
+rpc-http-tls-keystore-file="none.pfx"
+rpc-http-tls-keystore-password="none"
+rpc-http-tls-truststore-file="none.pfx"
+rpc-http-tls-truststore-password="none"
```

Comment:
do these need to be here if enabled is false?

Reply:
@rain-on  I think so, I followed example of `rpc-http-authentication-enabled=false`. I think this file is used by some sanity config file  unit test to validate all of the picocli options regardless whether they are enabled or disabled. We can set it to true as well.